cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -104,9 +104,9 @@ describe('Script Builder', () => {
         cy.wait(1200).then(() => {
           cy.reload()
           cy.getByTestID('flux-sync--toggle')
+          cy.getByTestID('flux-editor', {timeout: 30000})
```

Comment:
Pulled this flux-editor check into the `beforeEach()`, within each separate test context.
e.g. Results display tests (added here), schema browser tests (added below), and schema composition test (already added).